include/onnxruntime/core/platform/EigenNonBlockingThreadPool.h

```diff
@@ -132,7 +132,7 @@
 //
 // - When entering a parallel loop (or parallel section), a thread
 //   maintains a set of "preferred" worker hints, and initially
-//   submits tasks to these workers.  
+//   submits tasks to these workers.
```

Comment:
There are a lot of indentation / whitespace changes through this file.  I assume those are making it fit with current conventions we follow, so this is a one-time change rather than something that will get reverted?

I'll add specific comments below on code changes I noticed.  Could you let me know if I missed anything?  GitHub is not doing a great job of highlighting the changes

Reply:
These are the changes that are made by the VS formatter. Some previous runs failed Lint after the merge. So I had to format parts of the code. However, this is a one time change.